a3p-integration/.gitignore

```diff
@@ -15,6 +15,7 @@ upgrade-test-scripts
 # same for each proposal, an independent project
 proposals/*/.pnp.*
 proposals/*/.yarn/*
+proposals/a:upgrade-next/*
```

Comment:
I'm not sure about this one, but if you are, I'm still willing to approve.

Reply:
I'm still in the process of understanding this, as it appears to be intermittent. Sometimes artifacts are generated when the associated workflow runs, and other times they are not.